src/views/include/list/added.html

```diff
@@ -0,0 +1,44 @@
+{% if (add_team and add_team.length > 0)
+    or (add_repo and add_repo.length > 0)
+    or (add_member and add_member.length > 0) %}
+
+    <table class="govuk-table">
+        <caption class="govuk-table__caption govuk-table__caption--m">Added</caption>
+        <tbody class="govuk-table__body">
+            {% for team in add_team %}
+                <tr class="govuk-table__row">
+                    <th scope="row" class="govuk-table__header">Team</th>
+                    <td class="govuk-table__cell">{{ team.team_name }}</td>
+                    <td class="govuk-table__cell">
+                        <a class="govuk-link" href="/add-team/{{ team.id }}">
+                            Change or remove
```

Comment:
Change or remove in one link is a bit clunky, removing an entity requires two clicks. 

Could the functionality be split like in this example on [GDS Way (Sarah Philips)](https://design-system.service.gov.uk/components/summary-list/). 

It would be `Change | Remove`. Current implementation works, this is just an improvement suggestion.

Reply:
Nice improvement add a story in the backlog please